node.gyp

```diff
@@ -169,7 +169,8 @@
 
       'dependencies': [
         'node_js2c#host',
-        'deps/nghttp2/nghttp2.gyp:nghttp2'
+        'deps/nghttp2/nghttp2.gyp:nghttp2',
+        'node_postmortem_metadata#host'
```

Comment:
If you swap these two, you can reduce the diff by one line (because you don't need to append a comma to the nghttp2 dep.)